pages/apply/submit/declaration.tsx

```diff
@@ -40,6 +41,7 @@ const Declaration = (): JSX.Element => {
       }
 
       dispatch(completeApplication(application));
+      dispatch(createEvidenceRequest(application));
```

Comment:
I'm still not exactly sure with Redux thunks, but I have a feeling that maybe both of these posts are meant to go in the same thunk? So you could dispatch one action and then fire both of the requests from the one thunk handler (or have two handlers for the same action but that doesn't seem such a good idea).
This is probably absolutely fine, I can't see any benefit to be hand from changing it around at this stage.